### DISCUSSION TITLE
ResidentialModule01_Colony has an unopenable Custom Hatch

### DISCUSSION THREAD

**Discussion by MKMoose** [FakeFishGames/Barotrauma#15006] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

ResidentialModule01_Colony has a Custom Hatch in the top gap, at the top of the left ladder.

It's not an issue in the vanilla game since nothing ever generated there, or at least it never did before the Unto the Breach update, but custom outpost modules can change the layout. I've found it twice over the course of a campaign after adding a custom module.

The issue isn't 100% consistent in-game due to outpost generation seemingly randomizing which side's hatch gets removed when connecting modules.

### Reproduction steps

Open ResidentialModule01_Colony in the sub editor.
Look for the Custom Hatch at the top of the left ladder.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/6654c61e72a21dbe37281c0b2e8fb590438b53cb (commit in our private repo). Fix coming up in a future update.
